File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.UnitLibraryService.Models/save-unit-location_item.ts


import { ISaveUnitLocation_Item } from './isave-unit-location_item';

export class SaveUnitLocation_Item implements ISaveUnitLocation_Item {
  
	public CityID: number = 0;
	public Address1: string = "";
	public Address2: string = "";
	public Address3: string = "";
	public ModifiedByAppUserID: number = 0;
	public CountryID?: number;
	public StateID?: number;
  
}

File: GTTS/INL.UnitLibraryService.Models/SaveUnitLocation_Item.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace INL.UnitLibraryService.Models
{
    public class SaveUnitLocation_Item : ISaveUnitLocation_Item
    {
        public int CityID { get; set; }
        public string Address1 { get; set; }
        public string Address2 { get; set; }
        public string Address3 { get; set; }
        public int ModifiedByAppUserID { get; set; }
        public int? CountryID { get; set; }
        public int? StateID { get; set; }
    }
}
